dask/dataframe/io/tests/test_io.py

```diff
@@ -480,6 +481,28 @@ def test_from_dask_array_unknown_width_error():
         dd.from_dask_array(dx)
 
 
+@pytest.mark.gpu
+def test_from_array_cupy():
```

Comment:
For completeness, it'd be good to parametrize this test to cover the `cupy`->`cudf` dispatching as well as the `numpy`->`pandas` dispatching